bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/commands/bookies/RecoverCommand.java

```diff
@@ -103,6 +103,9 @@ private RecoverCommand(RecoverFlags flags) {
 
         @Parameter(names = {"-sku", "--skipunrecoverableledgers"}, description = "Skip unrecoverable ledgers")
         private boolean skipUnrecoverableLedgers;
+
+        @Parameter(names = { "-trb", "--throttleRateByBytes" }, description = "Throttle replication rate in bytes")
```

Comment:
```suggestion
        @Parameter(names = { "-rr", "--replicationrate" }, description = "Throttle replication rate in bytes")
```

Reply:
Maybe name it as `replicationrate` is more simple?
Just for consistency, looks like all of our parameters are in lower case.

Reply:
+1